tests/raft_scenarios/startup

```diff
@@ -0,0 +1,7 @@
+start_node
```

Comment:
This is a minimal scenario, to get the trace validation to line up, but it fails to match on "commit", for reasons that are not yet clear. Another bug became obvious along the way (#5798), but is not the root cause here. I think it may be the presence of the initial empty config that's at fault.

Reply:
It was.